features/validator.feature

```diff
@@ -0,0 +1,125 @@
+# Created by amnon at 12/04/2021
+Feature: file format validation
+  # see ticket #266
+  # (and see https://www.jetbrains.com/help/idea/handling-issues.html to make this
+  #  number clickable and leading to ZenHub)
+
+  Scenario Outline: 1.a filename extension matches the file format - fastq
+    Given a file with filename <file_name>
+    And format field set to <file_format>
+    When metadata is validated
+    Then File is <validation_state> after validation
+
+    Examples:
+      |           file_name |     file_format | validation_state |
+      |       file.fastq.gz |        fastq.gz |            VALID |
+      |          file.fq.gz |           fq.gz |            VALID |
+      |          file.fastq |           fastq |            VALID |
+      |   file.fastq.tar.gz |    fastq.tar.gz |            VALID |
```

Comment:
I don't think I've ever seen a tar fastq file but it doesn't hurt to test for it I guess!

Reply:
I've seen that in the configuration in the validator. That's why I added it as a test case. We don't need to actually test it. These unit tests should suffice.